xprof/Makefile.am

```diff
@@ -43,14 +43,20 @@ BTX_TIMELINE_GENERATED = \
 	btx_sink_timeline/btx_main.c
 
 BTX_AGGREG_GENERATED = \
-        btx_filter_aggreg/metababel/metababel.h \
-        btx_filter_aggreg/metababel/btx_component.h \
-        btx_filter_aggreg/metababel/btx_component.c \
-        btx_filter_aggreg/metababel/btx_upstream.h \
-        btx_filter_aggreg/metababel/btx_upstream.c \
-        btx_filter_aggreg/metababel/btx_downstream.h \
-        btx_filter_aggreg/metababel/btx_downstream.c \
-        btx_filter_aggreg/btx_main.c
+	btx_filter_aggreg/metababel/metababel.h \
+	btx_filter_aggreg/metababel/btx_component.h \
+	btx_filter_aggreg/metababel/btx_component.c \
+	btx_filter_aggreg/metababel/btx_upstream.h \
+	btx_filter_aggreg/metababel/btx_upstream.c \
+	btx_filter_aggreg/metababel/btx_downstream.h \
+	btx_filter_aggreg/metababel/btx_downstream.c \
+	btx_filter_aggreg/btx_main.c
+
+BTX_STRIPPER_GENERATED = \
+	btx_filter_aggreg/metababel/metababel.h \
```

Comment:
Shouldn't this be in it's own directory? Is potentially overwriting agg one or maybe not getting generated because is already generated for agg.

Reply:
Totally right! Fixed with add26c8